cli/go.mod

```diff
@@ -40,3 +40,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/cloudquery/plugin-sdk => ../../plugin-sdk
```

Comment:
You could `go get` an untagged 'version' of the SDK (using the `@commithash` syntax) and remove this line

Reply:
I'd rather not accidentally merge that in :)